extensions/amp-base-carousel/0.1/carousel.js

```diff
@@ -838,6 +869,8 @@ export class Carousel {
    * @private
    */
   resetScrollReferencePoint_(force = false) {
+    this.scrolling_ = false;
```

Comment:
Should this be after making sure that the user is not in the middle of a drag?

Reply:
I considered it, but we already have something to check for touching. So if they stopped scrolling (no scroll event recently), but they are still touching the screen, I think it would be cleaner to say that there is no scrolling, and interested code can check for touching as well. If they are dragging, we should get scroll events, so that would still count as scrolling.